pandas/core/groupby/groupby.py

```diff
@@ -767,10 +767,24 @@ def __iter__(self) -> Iterator[tuple[Hashable, NDFrameT]]:
         """
         Groupby iterator.
 
+        This method provides an iterator over the groups created by the ``resample``
+        or ``groupby`` operation on the object. The method yields tuples where
+        the first element is the label (group key) corresponding to each resampled
+        bin, and the second element is the subset of the data that falls within that
+        bin.
```

Comment:
`corresponding to each group or resampled bin` and `falls within that group or bin`.

Reply:
apologies for missing this. fixed now.